app.py

```diff
@@ -1,13 +1,129 @@
 #!/usr/bin/python3
-from flask import Flask, render_template
+from flask import Flask, render_template, jsonify, request
+import pymysql
+from flask_cors import CORS, cross_origin
+import json
 import uuid
+from datetime import datetime
+import time
 
-application = Flask(__name__)
+app = Flask(__name__)
+cors = CORS(app, resources={r"/*": {"origins": "*"}})
 
-@application.route("/")
+@app.route('/', strict_slashes=False)
 def mainpage():
     return render_template('login.html',
-                           cache_id=uuid.uuid4())
+                           cache_id=uuid.uuid4()
+    )
 
-if __name__ == "__main__":
-    application.run(host='0.0.0.0')
+@app.route('/save_exp/', methods=['GET', 'OPTIONS', 'POST'], strict_slashes=False)
+def save_exp():
+    """ inserts new experience into 'experiences' table """
+    response = request.data.decode('utf-8')
+    obj = json.loads(response)
+
+    con = pymysql.connect('localhost', 'wellness_dev', 'wellness_dev_pwd', 'wellness_dev_db')
```

Comment:
Currently, your suggestion only works if starting the app directly from the file. Gunicorn, however, has it's own environment and I haven't tied down how to do that yet. until I do, I'll leave it hardcoded. I will remove once the issue is resolved.

Reply:
You can set the env in your own enviornment as well when running app.py

Reply:
Maybe we’re talking about two diff things. I’m upstart, yes I can set the env variables but they are contained to the environment of the gunicorn managed process. When Python goes to read them, it’s reading the regular ones you see when typing env. And they don’t exist because Python it not accessing the ones set in the upstarts environment. I’ll try again later today. If it still does the same thing I’ll let you know.